src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      this.result += value;
+
+      return this;
```

Comment:
what happend if we remove `return this` here?

Reply:
Nothing😅, thanks, my bad